src/components/visualizations/chart/highcharts/customConfiguration.ts

```diff
@@ -360,6 +360,20 @@ function labelFormatter(config?: IChartConfig) {
     return formatLabel(this.y, get(this, 'point.format'), config);
 }
 
+export function percentageDataLabelFormatter(config?: IChartConfig) {
+    const isSingleAxis = get(this, 'series.chart.yAxis.length') !== 2;
```

Comment:
why not === 1 ?

Reply:
I have updated.
I suppose that chart will have on Y axis by default and this formatter only gets called by Highchart when there is at least one Y axis :)